mod/pages/views/default/resources/pages/edit.php

```diff
@@ -5,7 +5,7 @@
 
 $page_guid = (int) elgg_extract('guid', $vars);
 
-elgg_entity_gatekeeper($page_guid, 'object', 'page');
+elgg_entity_gatekeeper($page_guid, 'object', 'page', true);
 
 $page = get_entity($page_guid);
 if (!$page->canEdit()) {
```

Comment:
i believe the canEdit check can be removed now we have the check in the gatekeeper